sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -1134,6 +1134,12 @@ public abstract partial class ObjectSerializer
         public abstract System.Threading.Tasks.ValueTask SerializeAsync(System.IO.Stream stream, object? value, System.Type inputType, System.Threading.CancellationToken cancellationToken);
         public virtual System.Threading.Tasks.ValueTask<System.BinaryData> SerializeAsync(object? value, System.Type? inputType = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
     }
+    public abstract partial class SchemaValidator<T>
```

Comment:
Can you share context and motivation for adding this to the Azure.Core API?

Reply:
This primarily came from some architect feedback on our earlier designs. This serializer ended up being a lot more general that we initially anticipated, so in an effort to reflect that in the API, we were given feedback to move some pieces into Azure.Core. As a little bit of background, there are only three existing libraries that can generate JSON schemas from .NET types or validate .NET objects against JSON schemas. Two of them are written by hobby-ists, and one doesn't have an open-source compatible license. For this reason, we could not have built-in schema validation or generation in this package (which is different from our Avro serializer). Currently, writing our own implementation of this is not a worthwhile time investment.
My understanding is that by putting the abstract class in `Azure.Core`, we would leave the door open in case someone down the road decides to write an implementation of generate/validate. Then it would seamlessly integrate into our Schema Registry general serializer type, while still being usable by any developer looking to use JSON schema generation/validation in an SDK or otherwise.

Reply:
Got it - thank you so much for the helpful context!